extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/endpoints.html

```diff
@@ -29,10 +32,20 @@ <h5>REST Resources</h5>
  {/if}
  {/for}
  <h5>Static Resources</h5>
- {#for staticResources in info:staticResourcesInfo}
+ {#for resource in info:staticResourceInfo.resourceMap.get("/").children}
  <div class="row">
   <div class="col">
-   <a href="{staticResources}">{staticResources}</a>
+   {#if resource.isFolder}
+   <i class="far fa-folder-open" style="margin-right:5px; margin-top:5px"></i>
+   <a class="static" href="http://localhost:8080/q/dev/io.quarkus.quarkus-resteasy-reactive/view?path=/{resource.lastName}">
```

Comment:
You should use a relative path `view?...` not assume anything about the current host and port and path.

Reply:
Relative path meaning instead of `{resource.lastName}` ? 
Or you mean change this url http://localhost:8080/q/dev/io.quarkus.quarkus-resteasy-reactive/ to ...

Reply:
On putting `{urlbase}/view?path=/{resource.lastName}` it is taking me to this page -  http://localhost:8080/q/dev/io.quarkus.quarkus-resteasy-reactive/NOT_FOUND/view?path=/Images

![image](https://user-images.githubusercontent.com/46818757/107666916-10b35280-6cb5-11eb-86d7-152a8fd54899.png)

Reply:
Done :)
I am too quick in asking questions